src/io/read_tree_consistentrees_ascii.c

```diff
@@ -346,9 +381,12 @@ int64_t load_forest_ctrees(const int32_t forestnr, struct halo_data **halos, str
     }
 
 
-    /* Entire tree is loaded in. Fix upid's*/
-    const int max_snapnum = fix_upid(totnhalos, forest_halos, info, &(run_params->interrupted), verbose);
-
+    /* Entire tree is loaded in. Fix upid's (i.e., only keep 1-level halo hierarchy: FOF->subhalo */
+    const int max_snapnum = fix_upid(totnhalos, forest_halos, info, verbose);
+    if(max_snapnum < 0) {
+        ABORT(max_snapnum);
```

Comment:
:O An abort... For shame @manodeep ....

Reply:
And even more ABORTs! How are there so many ABORT statements still in the codebase!